app/src/main/res/layout/placeholder_item_card_tickets.xml

```diff
@@ -10,25 +10,37 @@
     app:cardCornerRadius="@dimen/card_elevation"
     app:cardElevation="@dimen/layout_margin_none">
 
-    <LinearLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="@dimen/avatar_medium"
         android:orientation="horizontal">
 
         <View
+            android:id="@+id/view3"
             android:layout_width="@dimen/card_width_45dp"
             android:layout_height="@dimen/card_width_45dp"
-            android:layout_marginLeft="@dimen/layout_margin_medium"
-            android:layout_marginTop="@dimen/layout_margin_medium"
-            android:background="@color/shimmer_background" />
+            android:layout_marginStart="8dp"
+            android:layout_marginLeft="8dp"
+            android:layout_marginTop="8dp"
```

Comment:
Define all dimensions in dimension resources file.

Reply:
Done..